source/Calamari.AzureAppService.Tests/TargetDiscoveryBehaviourIntegrationTestFixture.cs

```diff
@@ -288,6 +289,7 @@ private void CreateVariables(RunningDeployment context)
         ""roles"": [""{Role}""]
     }},
     ""authentication"": {{
+        ""type"": ""{Type}"",
```

Comment:
I've added this type to the authenticationDetails, is this a valid assumption that it'll come from server with a type?

Reply:
I think I added the `Type` property in server is just used by Telemetry but it probably doesn't hurt to add it in.